### ISSUE TITLE
Can this project be updated - SoftKey support and Feedback

### ISSUE THREAD

**Issue by tpk001** [bitfocus/companion-module-allenheath-avantis#10]
Can this project be updated to support full availability of supported options and for Companion v3 compatibility please?

**Comment by Bambii556** [comment]
Hi @tpk001 , i am busy upgrading this module now. it should have all this implemented

**Comment by josephdadams** [comment]
@Bambii556 what is your status on this? I don't want to take over upgrading it if you are in process/almost complete.

**Comment by mikecreates86** [comment]
Any update on this? I see that in December @Bambii556 you were working on this. Any ETA when it will make it to the beta?

**Comment by mattokay1987** [comment]
It will be awesome if this really gets implemented... :(

**Comment by jameskirsop** [comment]
It looks like @erlendjones has been doing some work, but a formal release on the repo would be helpful since it's now a few points behind what's represented in `package.json`.

**Comment by mikecreates86** [comment]
Checking in again on this- where is this project when it comes to being updated? Really think I'm not the only one using this console who would like to see it refreshed for all the features it could support.

**Comment by erlendjones** [comment]
Im afraid feedbacks are not possible as long as A&H won’t implement the needed API-Endpoints in the console.

**Comment by cmcrombie** [comment]
Has A&H been contacted for this request? It's strange that they would allow much of this on the SQ-5 and not for the Avantis.

**Comment by erlendjones** [comment]
Yes, they have, and they could not promise any improvements to the existing API.

**Comment by mikecreates86** [comment]
Any updates on this module? Firmware 2.0 has been out for a minute now and I'm having to use dLive modules to control the RackUltraFX settings. Can the Avantis module just be updated to work with all known controllable features?

**Comment by jumpy1066** [comment]
Speaking of which, the Avantis Ultra FX now has 96 channels. But right now, I can only select the 64 channels without the DPack—before the update. With the DPack, you have 96 channels.
